README.md

```diff
@@ -70,7 +70,7 @@ kubectl config use-context <name>
 Example first deployment to a local cluster (from the project root directory):
 ```  
  helm upgrade --install palisade . \
-  --set global.persistence.classpathJars.local.hostPath=$(pwd),global.persistence.dataStores.palisade-data-store.local.hostPath=$(pwd),global.persistence.kafka.local.hostPath=$(pwd),global.persistence.redisMaster.local.hostPath=$(pwd),global.persistence.redisSlave.local.hostPath=$(pwd),global.persistence.zookeeper.local.hostPath=$(pwd),traefik.install=true,kafka.install=true,redis.install=true,global.hosting=local,redis-cluster.install=false --timeout=200s
+  --set global.persistence.statefulset.pv.enabled=true,global.persistence.classpathJars.local.hostPath=$(pwd),global.persistence.dataStores.palisade-data-store.local.hostPath=$(pwd),global.persistence.kafka.local.hostPath=$(pwd),global.persistence.redisMaster.local.hostPath=$(pwd),global.persistence.redisSlave.local.hostPath=$(pwd),global.persistence.zookeeper.local.hostPath=$(pwd),traefik.install=true,kafka.install=true,redis.install=true,global.hosting=local,redis-cluster.install=false --timeout=200s
```

Comment:
Can this be split over multiple lines so the README is a bit easier to read 
```suggestion
    --set global.persistence.statefulset.pv.enabled=true \
    --set global.persistence.classpathJars.local.hostPath=$(pwd) \
    --set global.persistence.dataStores.palisade-data-store.local.hostPath=$(pwd) \
    --set global.persistence.kafka.local.hostPath=$(pwd) \
    --set global.persistence.redisMaster.local.hostPath=$(pwd) \
    --set global.persistence.redisSlave.local.hostPath=$(pwd) \
    --set global.persistence.zookeeper.local.hostPath=$(pwd) \
    --set traefik.install=true,kafka.install=true \
    --set redis.install=true \
    --set global.hosting=local \
    --set redis-cluster.install=false \
    --timeout=200s
```

Reply:
Thanks, changed it.